src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -278,9 +311,13 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
             else {
                 final ParseErrorMessage parseErrorMessage = result.getParseErrorMessage();
-                log(parseErrorMessage.getLineNumber(),
-                        parseErrorMessage.getMessageKey(),
-                        parseErrorMessage.getMessageArguments());
+                final boolean looseHtmlInJavadoc = "javadoc.unclosedHtml"
+                        .equals(parseErrorMessage.getMessageKey());
+                if (!looseHtmlInJavadoc || violateExecutionOnNonTightHtml) {
```

Comment:
We are to log violation for non-tight HTML when `javadoc.unclosedHtml` is the message key and `violateExecutionOnNonTightHtml` is true. Apart from that we are also to log violation for all other cases when message key is not `javadoc.unclosedHtml`. 

So basically this ensures that errors aren't logged when key is `javadoc.unclosedHtml` and  `violateExecutionOnNonTightHtml` is false.